.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# Contributing to `<media-chrome>`
+
+- [Questions](#questions)
+- [Bugs and Issues](#issues)
+- [Documentation Updates](#documentation)
+- [Feature Requests](#features)
+- [Submitting a Pull Request](#pull-requests)
+- [Merging a Pull Request](#merging)
+- [Releasing](#releasing)
+
+## <a name="questions">Questions</a>
+
+Have a question? Want to start a discussion? For now, you can simply [Open a New Discussion](https://github.com/muxinc/media-chrome/discussions/new), and choose a fitting category and title.
+
+## <a name="issues">Bugs and Issues</a>
+
+If you think you've found a bug, make sure you review and fill out a [Bug Report](https://github.com/muxinc/media-chrome/issues/new/choose) before starting any work. This will ensure for both yourself and the maintainers that the issue in question can be properly confirmed, reproduced, smoke tested, etc. Once done, go ahead and follow our [Submitting a Pull Request](#pull-requests) guide.
```

Comment:
```suggestion
If you think you've found a bug, make sure you review and fill out a [Bug Report](https://github.com/muxinc/media-chrome/issues/new/choose) before starting any work. This will ensure for both yourself and the maintainers that the issue in question can be properly confirmed, reproduced, smoke tested, etc. Once done, if you want to try to fix the issue yourself, go ahead and follow our [Submitting a Pull Request](#pull-requests) guide. Contributions are welcome and encouraged!
```